packages/tasit-sdk/package.json

```diff
@@ -16,12 +17,21 @@
     "url": "git+https://github.com/tasitlabs/tasitsdk.git"
   },
   "scripts": {
-    "test": "echo \"Error: run tests from root\" && exit 1"
+    "prepublish": "npx babel src --out-dir lib --ignore **/*.test.js",
```

Comment:
@marcelomorgado  "As of npm@5, `prepublish` scripts are deprecated. Use `prepare` for build steps and `prepublishOnly` for upload-only."